com.microsoft.mrtk.input/Utilities/SpeechUtils.cs

```diff
@@ -11,12 +11,12 @@ namespace Microsoft.MixedReality.Toolkit.Input
     public static class SpeechUtils
     {
         /// <summary>
-        /// Gets the first running <see cref="PhraseRecognitionSubsystem"/> instance.
+        /// Gets the first running <see cref="KeywordRecognitionSubsystem"/> instance.
         /// </summary>
-        /// <returns>The running <see cref="PhraseRecognitionSubsystemInstance"/>, or null.</returns>
-        public static PhraseRecognitionSubsystem GetSubsystem()
+        /// <returns>The running <see cref="KeywordRecognitionSubsystemInstance"/>, or null.</returns>
+        public static KeywordRecognitionSubsystem GetSubsystem()
         {
-            return XRSubsystemHelpers.GetFirstRunningSubsystem<PhraseRecognitionSubsystem>() as PhraseRecognitionSubsystem;
+            return XRSubsystemHelpers.GetFirstRunningSubsystem<KeywordRecognitionSubsystem>() as KeywordRecognitionSubsystem;
```

Comment:
nit: is the `as KeywordRecognitionSubsystem` part needed? I thought this helper returned the same type as the passed-in `T`

Reply:
I also wonder if we want to move this to the new pattern @Zee2 is introducing for the hands aggregator in #11333: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/11333/files#diff-6bb0070ef5fe1b4e52210a345ab2f2c2f693e0b9cff69e18e05456aef8969376R160

Basically, deprecating the old helper from `HandsUtils` and putting it into `XRSubsystemHelpers` directly

Reply:
Fixed!